rpm/cray/csm/noos/index.yaml

```diff
@@ -43,13 +43,14 @@ https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/noos/:
     - csm-node-identity-1.0.22-1.noarch
     - csm-ssh-keys-1.5.6-1.noarch
     - csm-ssh-keys-roles-1.5.6-1.noarch
-    - csm-testing-1.16.52-1.noarch
-    - goss-servers-1.16.52-1.noarch
+    - csm-testing-1.17.5-1.noarch
+    - goss-servers-1.17.5-1.noarch
     - hpe-csm-goss-package-0.3.21-hpe3.x86_64
-    - hpe-csm-scripts-0.5.7-1.noarch
+    - hpe-csm-scripts-0.6.2-1.noarch
     - hpe-yq-4.33.3-1.aarch64
     - hpe-yq-4.33.3-1.x86_64
     - ilorest-4.2.0.0-20.x86_64
+    - iuf-cli-1.6.1-1.x86_64
```

Comment:
I believe the 1.5.z version of `iuf-cli` are intended for CSM 1.5, and the 1.6.z versions are intended for CSM 1.6. I'm not sure what all the differences are between the latest 1.5.z and 1.6.z releases of `iuf-cli`, but there are definitely some. At the very least my change for CASMINST-6518 will be in 1.6.2, but I do not plan to backport to a 1.5.z release. There are branches corresponding to CSM releases in the iuf-cli repo. I didn't set it up this way. I'm just reporting what I've learned from talking with @leliasen-hpe.

Reply:
I have not backported iuf-cli to release/1.5 yet, and made this backport prematurely.

Reply:
Thanks for clarifying. I started that process here: https://github.com/Cray-HPE/iuf-cli/pull/162

Reply:
Rusty updated the PR to pull in iuf-cli-1.5.5 instead, so this is resolved.